keyboards/lck75/oled.c

```diff
@@ -0,0 +1,141 @@
+/* Copyright 2020 @wafflekeebs/@waffle#6666

```

Comment:
Ideally, this should be included in the keyboard's c file, and the `oled_task_user` defined weakly (eg `__attribute__((weak)) void oled_task_user(void) {`

Reply:
id like to keep the copyright there as it's my friend that helped with the OLED code for the board. edited the `oled_task_user`